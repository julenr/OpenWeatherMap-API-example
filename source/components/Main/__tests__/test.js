import React from 'react';
import renderer from 'react-test-renderer';

import Cities from '..';

describe('Range component', () => {
  const data = {
    ciudad1: ['1', '2'],
    ciudad2: ['1', '2'],
    ciudad3: ['1', '2']
  };
  test('Snapshot', () => {
    const component = renderer.create(
      <Cities data={data} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
