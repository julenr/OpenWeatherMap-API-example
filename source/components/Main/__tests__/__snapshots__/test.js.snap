// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Range component Snapshot 1`] = `
<div
  className="sc-bdVaJa jFJYjJ"
>
  <div
    className="sc-bwzfXH kPisAc"
  >
    <div
      className="sc-htpNat jgHHHu"
    >
      ciudad1
    </div>
    <div
      className="sc-bxivhb fBbAJl"
    >
      1
      ˚
    </div>
    <div>
      Histórico
    </div>
    <div
      className="sc-ifAKCX bwqiFx"
    >
      <div>
        1
        ˚
      </div>
      <div>
        2
        ˚
      </div>
    </div>
  </div>
  <div
    className="sc-bwzfXH kPisAc"
  >
    <div
      className="sc-htpNat jgHHHu"
    >
      ciudad2
    </div>
    <div
      className="sc-bxivhb fBbAJl"
    >
      1
      ˚
    </div>
    <div>
      Histórico
    </div>
    <div
      className="sc-ifAKCX bwqiFx"
    >
      <div>
        1
        ˚
      </div>
      <div>
        2
        ˚
      </div>
    </div>
  </div>
  <div
    className="sc-bwzfXH kPisAc"
  >
    <div
      className="sc-htpNat jgHHHu"
    >
      ciudad3
    </div>
    <div
      className="sc-bxivhb fBbAJl"
    >
      1
      ˚
    </div>
    <div>
      Histórico
    </div>
    <div
      className="sc-ifAKCX bwqiFx"
    >
      <div>
        1
        ˚
      </div>
      <div>
        2
        ˚
      </div>
    </div>
  </div>
</div>
`;
